This patch (a) fixes a segfault on arm64 by aligning `longopts_help`'s length
with `longopts`', and (b) fixes help output (which is otherwise misaligned
after `--notify`). This has been submitted upstream in
https://github.com/OpenRC/openrc/pull/860
--
diff --git a/src/start-stop-daemon/start-stop-daemon.c b/src/start-stop-daemon/start-stop-daemon.c
index dc67c126..05e28aa4 100644
--- a/src/start-stop-daemon/start-stop-daemon.c
+++ b/src/start-stop-daemon/start-stop-daemon.c
@@ -171,6 +171,7 @@ const char * const longopts_help[] = {
 	"Print dots each second while waiting",
 	"Set process scheduler",
 	"Set process scheduler priority",
+	"Configures experimental notification behaviour",
 	longopts_help_COMMON
 };
 const char *usagestring = NULL;
